Mathlib/Order/Filter/AtTopBot.lean

```diff
@@ -1545,7 +1543,7 @@ theorem prod_atTop_atTop_eq [Preorder α] [Preorder β] :
 #align filter.prod_at_top_at_top_eq Filter.prod_atTop_atTop_eq
 
 -- Porting note: generalized from `SemilatticeSup` to `Preorder`
-theorem prod_atBot_atBot_eq [Preorder β₁] [Preorder β₂] :
+theorem prod_atBot_atBot_eq {β₁ β₂ : Type*} [Preorder β₁] [Preorder β₂] :
```

Comment:
Let's just use `α` and `β` here to match the rest of the section?

Reply:
Done; and also re-used alpha and beta in a few more places.